features/topology-map/plugins/org.opennms.features.topology.plugins.topo.history/src/test/java/org/opennms/features/topology/plugins/topo/BundleContextHistoryManagerTest.java

```diff
@@ -258,10 +369,15 @@ private void setBehaviour(GraphContainer graphContainerMock) {
         		.andReturn(selectionManagerMock)
         		.anyTimes();
 
+        Criteria[] criteria = new Criteria[availableCriteria.size()];
+
         EasyMock
                 .expect(graphContainerMock.getCriteria())
-                .andReturn(new Criteria[0])
+                .andReturn(availableCriteria.toArray(criteria))
```

Comment:
`capturedCriteria` object was introduced to intercept parameters from the method `addCriteria`. Objects from the `capturedCriteria` are later used in assertion.
If the method `getCriteria` returns the same values, then the comparison will always return true. The very idea of test will then be lost.
Besides, at the beginning the `capturedCriteria` is empty and the `getCriteria` method is called earlier than `addCriteria`